packages/p2p-v2/package.json

```diff
@@ -8,7 +8,7 @@
     },
     "scripts": {
         "analyze:stats": "NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.js\" --profile --json=stats.json",
-        "analyze:build": "webpack-bundle-analyzer --no-open -m static -r analyzed.html stats.json ./dist && webpack-bundle-analyzer -m json stats.json ./dist",
+        "analyze:build": "webpack-bundle-analyzer --no-open -m static -r treemap.html stats.json ./dist && webpack-bundle-analyzer -m json stats.json ./dist",
```

Comment:
p2p-v2 package is being removed as we discussed, I believe the changes here are not needed.